docs/install.md

```diff
@@ -92,6 +93,20 @@ If you wish to install an alternative gadget image, you could use the following
 $ kubectl gadget deploy --image=ghcr.io/myfork/inspektor-gadget:tag
 ```
 
+### Using the bcc image
+
+You can use the bcc image of Inspektor Gadget by using the `--bcc` flag:
```

Comment:
Perhaps add some more context about this BCC image:

"The BCC image doesn't require the kernel to support BTF, however it's able to work on older versions. Notice that this support is deprecated and will be removed soon".